README.md

```diff
@@ -1,3 +1,50 @@
-# Pajarito
+# Pajarito (Polyhedral Approximation in Julia: Automatic Reformulations for InTeger Optimization)
+
+Pajarito is a mixed-integer convex programming solver package in [Julia](http://julialang.org/) that currently implements an out-of-the-box polyhedral outer approximation algorithm under src/nonlinear.jl and a polyhedral outer approximation and branch and cut for mixed-integer conic programming algorithm under src/conic.jl.
 
 [![Build Status](https://travis-ci.org/mlubin/Pajarito.jl.svg?branch=master)](https://travis-ci.org/mlubin/Pajarito.jl) [![codecov.io](https://codecov.io/github/mlubin/Pajarito.jl/coverage.svg?branch=master)](https://codecov.io/github/mlubin/Pajarito.jl?branch=master)
+
+## Installation
+
+Pajarito can be installed through Julia:
+
+```
+julia> Pkg.clone("https://github.com/mlubin/Pajarito.jl.git")
+```
+
+It is recommended to install [ConicNonlinearBridge.jl](https://github.com/mlubin/ConicNonlinearBridge.jl) for the conic programming subproblems in case of numerical instability. See test/ folder for examples.
+
+## Supported solvers
+
+Pajarito requires a mixed-integer linear programming solver and a nonlinear programming solver for src/nonlinear.jl or a conic solver for src/conic.jl. 
+
+For MILP solver, it currently supports only [CPLEX](http://www-01.ibm.com/software/commerce/optimization/cplex-optimizer/) and [GuRoBi](http://www.gurobi.com/) for the branch and cut algorithm for mixed-integer conic programming problems. Pajarito further supports [GLPK](http://www.gnu.org/software/glpk/) for polyhedral outer approximation under both src/nonlinear.jl and src/conic.jl. MILP solver can be specified through the `mip_solver` flag, i.e. `PajaritoConicSolver(mip_solver=CplexSolver())`.
+
+For NLP solver, Pajarito has been tested with [Ipopt](https://projects.coin-or.org/Ipopt), [KNITRO](http://www.ziena.com/knitro.htm). NLP solver can be specified through `nlp_solver` flag, i.e. `PajaritoSolver(nlp_solver=IpoptSolver())`.
+
+For Conic solver, Pajarito has been tested with [SCS](https://github.com/cvxgrp/scs), [ECOS](https://github.com/ifa-ethz/ecos), and furthermore ConicNonlinearBridge for flexible choice of a supported NLP solver (Ipopt and KNITRO). Conic solver can be specified through `conic_solver` flag, i.e. `PajaritoConicSolver(conic_solver=KnitroSolver())`.
+
+All solvers can have their parameters specified through their corresponding Julia interfaces.
+
+## Supported problem classes
+
+Pajarito supports mixed-integer smooth convex programming problems through [JuMP.jl](https://github.com/JuliaOpt/JuMP.jl) (this is a reimplementation of [Bonmin](https://projects.coin-or.org/Bonmin) on Julia) and mixed-integer disciplined convex programming problems through [Convex.jl](https://github.com/JuliaOpt/Convex.jl).
```

Comment:
"this is a reimplementation of [Bonmin](https://projects.coin-or.org/Bonmin) on Julia" should be "this functionality is similar to that of [Bonmin](https://projects.coin-or.org/Bonmin)"